scene/gui/graph_edit.cpp

```diff
@@ -402,6 +415,8 @@ void GraphEdit::add_child_notify(Node *p_child) {
 	if (gn) {
 		gn->set_scale(Vector2(zoom, zoom));
 		gn->connect("offset_changed", this, "_graph_node_moved", varray(gn));
+		gn->connect("selected", this, "_graph_node_selected", varray(gn));
```

Comment:
I don't see method bindings in `_bind_methods` which is necessary to connect callbacks by string, so this wouldn't work.

Reply:
Fixed